edelphi/src/main/java/fi/metatavu/edelphi/pages/panel/admin/report/thesis/AbstractThesisScale2DQueryReportPage.java

```diff
@@ -122,9 +140,9 @@ protected Chart createBarChart(ChartContext chartContext, QueryPage queryPage, S
     // TODO These could be calculated elsewhere and added below the chart image?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)